src/bin/tikv-server.rs

```diff
@@ -919,10 +1039,24 @@ fn run_raft_server(pd_client: RpcClient,
         .unwrap_or_else(|err| exit_with_err(format!("{:?}", err)));
     let trans = server.transport();
 
+    // Create raft_cf engine.
+    let raft_rocksdb_dir = get_raft_rocksdb_data_dir(&cfg.storage.data_dir, config);
```

Comment:
old rocksdb has 4 cfs, but we use 3 cfs now, so the Open will fail, @zhangjinpeng1987 ?